Monika After Story/game/script-topics.rpy

```diff
@@ -14195,3 +14195,28 @@ label monika_translating_poetry:
         m 5ekbsa "It would be nice if you could read some of them for me sometime..."
         m 5rkbsu "You'd have to translate them for me first, though~"
     return
+init 5 python:
+    addEvent(
+        Event(
+            persistent.event_database,
+            eventlabel="monika_diving",
+            category=["nature"],
+            prompt="Diving",
+            random=True
+        )
+     )
+label monika_diving:
+    m 3eub "[player], have you heard about diving?"
+    m 1eub "this is scuba diving with special equipment."
+    m 3wub "It's very exciting. Just imagine you are diving."
```

Comment:
Instead of telling the player to imagine diving, it might be better to paint a picture in their head and tell them to imagine that. For example, describing some of the things you see while scuba diving such as fish or reefs. I think this helps better explain what it is, while also describing the experience.

Reply:
As for this, I have no ideas.